charts/node/templates/statefulset.yaml

```
@@ -699,7 +715,7 @@ spec:
               name: chain-data
             - mountPath: /keystore
               name: chain-keystore
-            {{- if and .Values.node.isParachain (not .Values.node.externalRelayChain.enabled)}}
+            {{- if and .Values.node.isParachain (and (not .Values.node.collatorExternalRelayChain.enabled ) (not .Values.node.collatorLightClient.enabled)) }}
```

you can avoid having nested `add`, the `add` function [supports](https://helm.sh/docs/chart_template_guide/function_list/#and) multiple arguments:
```suggestion
            {{- if and .Values.node.isParachain (not .Values.node.collatorExternalRelayChain.enabled ) (not .Values.node.collatorLightClient.enabled) }}
```